pkgs/development/python-modules/pyside/shiboken.nix

```diff
@@ -18,7 +18,6 @@ buildPythonPackage rec {
   pname = "pyside-shiboken";
   version = "1.2.4";
   format = "other";
-  disabled = !isPy3k;
```

Comment:
Please put this into a separate commit. Is this an accident or did you test that it now works for python3?

Reply:
I don't know whether it works in Python3. I just found out it is required as a dependency of Natron that requires Python2 and it works out of the box.

Extracted into #115081.